applications/portal/backend/api/tests/test_antibodies.py

```diff
@@ -108,14 +109,14 @@ def test_create(self):
         a.save()
 
         duplicated = AddAntibodyDTO(**example_ab)
-        # duplicated.vendorName = "My vendor synonym" # should keep the same vendor and add a synonym
```

Comment:
removing this because - otherwise -  it won’t be a duplicate 

condition for duplicate is both vendor and catalog_num should be same.
![image](https://github.com/MetaCell/scicrunch-antibody-registry/assets/59233227/f190fe85-2a71-43ea-bcfc-b727d70a8409)